### PR TITLE
Initial billing interface pull request / code review

### PR DESCRIPTION
**Pull request by dylanlott**
This is an initial pull request to start getting bridge-gui up and running with the billing interface and server. 

`storj-lib` is purposefully rolled back to storj@2.0.0 for compatibility reasons. If this is going to cause any issues, let me know, but this was a faster way to get this deployed than writing the library necessary for gui to talk to bridge-client. In the future, we should create a browser-only version of storj-lib. I talked with both @retrohacker and @bookchin about this and this seems to be the logical direction that `storj-lib` is heading, anyway.

### CODE REVIEW

File: .bash_history [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ls
```
Comment (retrohacker) [inline .bash_history]: Why is this committed?

File: dockerfiles/billing-develop.dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM node:4.5
+
+RUN apt-get update
+RUN apt-get install -y git
```
Comment (retrohacker) [inline dockerfiles/billing-develop.dockerfile]:
```Dockerfile
RUN apt-get update \
 && apt-get install -y --force-yes git \
 && rm -rf /var/lib/apt/lists/*
```
No need to keep the apt cache around

File: dockerfiles/billing-develop.dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN mkdir /billing
+WORKDIR /billing
```
Comment (retrohacker) [inline dockerfiles/billing-develop.dockerfile]: `WORKDIR` will make the directory for you https://docs.docker.com/engine/reference/builder/#/workdir

File: dockerfiles/billing-develop.dockerfile [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+ADD ./dockerfiles/files/billing.config.develop.json /billing/billing.config.develop.json
+RUN mkdir -p $HOME/.storj-billing/config
```
Comment (retrohacker) [inline dockerfiles/billing-develop.dockerfile]: `ADD` will create the directory for you: https://docs.docker.com/engine/reference/builder/#/add

File: dockerfiles/bridge-develop.dockerfile [hunk 5]
```diff
@@ -0,0 +1,20 @@
+FROM node:4.5
+
+RUN apt-get update
```
Comment (retrohacker) [inline dockerfiles/bridge-develop.dockerfile]: Same comment as above

File: dockerfiles/bridge-develop.dockerfile [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+RUN apt-get install -y git
+
+RUN mkdir /bridge
```
Comment (retrohacker) [inline dockerfiles/bridge-develop.dockerfile]: Same comment as above

File: dockerfiles/bridge-develop.dockerfile [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+WORKDIR /bridge
+
+ADD ./dockerfiles/files/bridge.config.develop.json /bridge/bridge.config.develop.json
+RUN mkdir -p $HOME/.storj-bridge/config
```
Comment (retrohacker) [inline dockerfiles/bridge-develop.dockerfile]: Same comment as above

File: dockerfiles/bridge-gui-development.dockerfile [hunk 8]
```diff
@@ -0,0 +1,16 @@
+FROM node:6
+
+EXPOSE 3001
+
+RUN mkdir /bridge-gui
```
Comment (retrohacker) [inline dockerfiles/bridge-gui-development.dockerfile]: Same comment as above

File: makefile [hunk 9]
```diff
@@ -0,0 +1,28 @@
+.SILENT:
+.DEFAULT:
+
+MAKEFILES_DIR = makefiles
+
+help:
+	if [ $(firstword $(MAKECMDGOALS)) == $@ ]; then \
+		echo 'Run `make <command> help` for more specific help' | fold -s; \
```
Comment (retrohacker) [inline makefile]: WOAH! `fold` is super clever!

File: makefile [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+		echo ''; \
+		echo 'commands:'; \
+		for file in $(wildcard $(MAKEFILES_DIR)/*.makefile); do \
```
Comment (retrohacker) [inline makefile]: Hmm, I don't have a `wildcard` on debian, and aptitude isn't showing any results.

If I understand intent, wouldn't this work with a simple `for file in "${MAKEFILES_DIR}/*.makefile"; do`

File: .bash_history (hunk 1 above)
Comment (dylanlott) [inline .bash_history]: Removed this, not even sure how it got in there to begin with.

File: dockerfiles/billing-develop.dockerfile (hunk 2 above)
Comment (dylanlott) [inline dockerfiles/billing-develop.dockerfile]: Updated

File: dockerfiles/billing-develop.dockerfile (hunk 3 above)
Comment (dylanlott) [inline dockerfiles/billing-develop.dockerfile]: Updated

**Pull request closed without merging**